gratipay/email.py

```diff
@@ -86,9 +86,10 @@ def put(self, to, template, _user_initiated=True, **context):
                     raise Throttled()
 
 
-    def flush(self):
+    def flush(self, _flush_one=None):
         """Load messages queued for sending, and send them.
         """
+        flush_one = _flush_one or self._flush_one
```

Comment:
If the aim is to empty the email queue in tests, might be clear to use a separate 'email_queue.purge()' function. (Tests shouldn't have to be aware of the fact that we purge one email at a time)